releasetasks/templates/tc_update_verify.yml.tmpl

```diff
@@ -0,0 +1,51 @@
+{% set uv_buildername = "{}_{}_update_verify".format(platform, branch) %}
+{% set uv_totalchunks = 12 %}
```

Comment:
If I am right about release_graph.yml call, I think we need to make the total chunks here 6